Classes/Command/WarmupCommand.php

```diff
@@ -61,7 +61,7 @@ public function execute(InputInterface $input, OutputInterface $output)
         foreach ($extensionKeys as $extensionKey) {
             $output->write($extensionKey . ':');
             $results = $service->warmup($extensionKey)[$extensionKey];
-            $numberOfTemplates = count($results['results']) ?: 0;
+            $numberOfTemplates = count($results['results'] ?? 0);
```

Comment:
Thanks for the patch @JBlocks - but this also causes a warning because if `$results['results']` is `null` then the resulting expression is `count(0)` which triggers a warning about not countable.

Reply:
fixed it with a new patch